### DISCUSSION TITLE
Show pictures in a picture view - is this gone?

### DISCUSSION THREAD

**Discussion by knezi** [gsantner/markor#2039] [Q&A]
I can see functionality described in https://github.com/gsantner/markor/issues/365 merged into master, but it doesn't seem to be present in the current code. Has this been later omitted or is it still somewhere?

**Comment by gsantner** [comment]
You can click a file in the file manager and Markor will show it, includling additional information and metadata.

**Reply by knezi** [reply]
I mean what you described here https://github.com/gsantner/markor/pull/395

It's about opening images linked in a file from the text editor.
